content/projects/tdd/password-checker/part1/_index.md

```diff
@@ -149,4 +149,11 @@ Please don't re-implement the same check in two different places. Good code is D
 
 ## Java Resources
 
-- https://howtodoinjava.com/junit5/expected-exception-example/
\ No newline at end of file
+- https://howtodoinjava.com/junit5/expected-exception-example/
+
+## Instructions for reviewers
+
+- For passwordStrength function, If any one of these passwords is passed in `P@ssw 12` or `User1@` ensure Invalid is returned.
```

Comment:
Just 1 small thing: For passwordStrength function, `If` - the I should not be a capital